lms/static/my.css

```diff
@@ -12,19 +21,19 @@ div#code-view {
   text-align: left;
 }
 
-::-webkit-input-placeholder {
+:lang(he) ::-webkit-input-placeholder {
```

Comment:
I prefer not to use the `:lang` pseudo class. It forcefully chooses the language for the user.

Use regular class like `.rtl-language` and put the conditions whether or not to append this class in the HTML